controllers/module_reconciler.go

```diff
@@ -118,6 +124,18 @@ func (r *ModuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	r.setKMMOMetrics(ctx)
 
+	if mod.Spec.ModuleLoader.ServiceAccountName == "" {
+		logger.Info("Handle default driver container ServiceAccount")
+		if err := r.handleDriverContainerServiceAccount(ctx, mod); err != nil {
+			return res, fmt.Errorf("could not handle default driver container ServiceAccount: %w", err)
```

Comment:
```suggestion
		logger.Info("Handle default module loader ServiceAccount")
		if err := r.handleModuleLoaderServiceAccount(ctx, mod); err != nil {
			return res, fmt.Errorf("could not handle default module loader ServiceAccount: %w", err)
```

Reply:
Refactored the internal RBAC API and changed all `driver container` references to `module loader` ones.